test/python/pointcloud2_test.py

```diff
@@ -53,4 +53,33 @@
 assert str(
     read_points[1]) == "Point(x=1.0, y=2.0, z=3.0, radar_relative_radial_velocity=4.0, signal_to_noise_ratio=5.0, ground_relative_radial_velocity=nan)", "Got:" + str(read_points[1])
 
+print("Success reading PointCloud2 data")
+
+
+print("Creating a PointCloud2 embedding entities... ")
+entities = [[99, 4, 20.5, -2.0, 1.0, 10.2, 25.0, 0, 0, 0, 1, 2, 5, 2, 254, 254],
+            [100, 2, 10.0, 2.0, 1.0, -10.0, 0.0, 0, 0, 0, 1, float('nan'), float('nan'), float('nan'), 254, 12]]
+
+entities_cloud = provizio_dds.point_cloud2.make_radar_entities(provizio_dds.point_cloud2.make_header(10, 20, "test_entities"), entities)
+
+# Check the entities cloud
+print("Checking the embedded entities in PointCloud2 metadata...")
+assert entities_cloud.fields()[0].name() == "entity_id"
+assert entities_cloud.fields()[1].name() == "entity_class"
+assert entities_cloud.fields()[2].name() == "x"
+assert entities_cloud.fields()[3].name() == "y"
+assert entities_cloud.fields()[4].name() == "z"
+assert entities_cloud.fields()[5].name() == "radar_relative_radial_velocity"
+assert entities_cloud.fields()[6].name() == "ground_relative_radial_velocity"
+assert entities_cloud.fields()[7].name() == "orientation"
+assert entities_cloud.fields()[8].name() == "size"
+assert entities_cloud.fields()[9].name() == "entity_confidence"
+assert entities_cloud.fields()[10].name() == "entity_class_confidence"
+
+# Read it
+print("Reading the PointCloud2 entities and checking its data...")
+
+read_entities = provizio_dds.point_cloud2.read_points(entities_cloud)
```

Comment:
@ClementBotty It'll likely work, but it doesn't fix the issue you releaved in the previous version of that tests. I'm OK not to do it in this PR, but please make a Jira bug.